<launch>
  <arg name="nr" default="1" />
  <node name="car" pkg="car" type="car_node" output="screen">
    <param name="car_num" value="$(eval str(arg('nr')))" />
  </node>
</launch>
